examples/logs-basic-in-memory/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
```

Comment:
I moved the example to examples/ in https://github.com/open-telemetry/opentelemetry-rust/tree/main/opentelemetry-appender-log.

I hope that's okay if not, I guess it could be removed as it's in the comments anyways.